book/src/SUMMARY.md

```diff
@@ -1,11 +1,30 @@
 # Table of contents
 
-* [The Solana Codebase](introduction.md)
-* [Building from Source](getting-started/README.md)
-  * [Example Client: Web Wallet](getting-started/webwallet.md)
-* [Programming Model](programs/README.md)
-  * [Example: Tic-Tac-Toe](programs/tictactoe.md)
-  * [Drones](programs/drones.md)
+* [Introduction](introduction.md)
+* [Using Solana](using-solana.md)
+  * [Command-line Client](cli.md)
+  * [Paper Wallet](paper-wallet/README.md)
+    * [Installation](paper-wallet/installation.md)
+    * [Paper Wallet Usage](paper-wallet/usage.md)
+  * [Offline Signing](offline-signing/README.md)
+    * [Durable Transaction Nonces](offline-signing/durable-nonce.md)
+* [Developing Applications](apps/README.md)
+  * [Example: Web Wallet](apps/webwallet.md)
+  * [Example: Tic-Tac-Toe](apps/tictactoe.md)
+  * [Drones](apps/drones.md)
+  * [Anatomy of a Transaction](transaction.md)
+  * [JSON RPC API](apps/jsonrpc-api.md)
+  * [JavaScript API](apps/javascript-api.md)
+* [Running a Validator](running-validator/README.md)
+  * [Validator Requirements](running-validator/validator-reqs.md)
+  * [Choosing a Testnet](running-validator/validator-testnet.md)
+  * [Installing the Validator Software](running-validator/validator-software.md)
+  * [Starting a Validator](running-validator/validator-start.md)
+  * [Staking](running-validator/validator-stake.md)
+  * [Monitoring a Validator](running-validator/validator-monitor.md)
+  * [Publishing Validator Info](running-validator/validator-info.md)
+  * [Troubleshooting](running-validator/validator-troubleshoot.md)
+* [Running an Archiver](running-archiver.md)
 * [A Solana Cluster](cluster/README.md)
```

Comment:
Name isn't too obvious.. How about "Understand Solana's Architecture" to match the "verb noun" naming convention we are using now?

Reply:
Fixed in bb840d4